### DISCUSSION TITLE
Thoughts on (future) compatibility for PieFed and/or Sublinks?

### DISCUSSION THREAD

**Discussion by nightgolfer** [aeharding/voyager#1343] [Ideas]
My home instance is Beehaw, and, as you might be aware, they're thinking very hard about finding a new platform because of irreconcilable differences with the Lemmy dev team. Hopefully, they'll stay in the Fediverse, on something using ActivityPub.

Although there's no way to know where they end up going, I know PieFed and Sublinks (although neither has launched yet) are both in the Consideration pile.

@aeharding have you thought about the possibility of adding support for either in the future? While I think both are going to be Lemmy compatible, I'm sure they're quite different in many ways, and it might just be too much of a headache, but I'm turn between my love for Voyager and Beehaw, and hoping there's a slim chance I can keep both.

Bonus question — Beehaw seems to be serious enough about leaving Lemmy that they aren't planning to update their server beyond Lemmy 0.18, which means that I can't use Voyager beyond the cutoff for 0.18 support, which is today. I don't know enough about PWAs, but I get the feeling that I won't be able to "downgrade" Voyager to a version that still has 0.18 support (right?). Is it possible to install an older version of the Native app (Android, specifically), or is that a moot point because the Native apps are somehow just wrappers for the PWA? 🤔

![Screenshot_2024-03-18-11-55-31-85_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/aeharding/voyager/assets/32328229/04bd92b6-d292-450d-99b7-b555e9b6eb8f)

**Comment by aeharding** [comment]
> Is it possible to install an older version of the Native app (Android, specifically), or is that a moot point because the Native apps are somehow just wrappers for the PWA? 🤔

Yes, definitely, you can install the latest Voyager 1.x release from Github Releases and continue to use with Lemmy 0.18. But you will not get any new features.

> have you thought about the possibility of adding support for either in the future?

It is too early to tell. :) They are both good projects, and as they mature Voyager might add support. However, in the immediate future, they will remain unsupported.

**Reply by nightgolfer** [reply]
Thank you! I'll switch from PWA to Native Android 😊👍

Completely understand with it being early days with the new platforms, but very happy that you're at least open to the idea 😃👌
